src/commands/mail/address/update.tsx

```diff
@@ -0,0 +1,267 @@
+    },
+  ];
+
+  protected async setAddress(
+    mailAddressId: string,
+    address: string,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the address of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressAddress({
+        mailAddressId: mailAddressId,
+        data: {
+          address: address,
+        },
+      });
+
+      assertStatus(response, 204);
+    });
+    await process.complete(
+      <Success>
+        The address of your mail address was successfully updated.
+      </Success>,
+    );
+  }
+
+  protected async setCatchAll(
+    mailAddressId: string,
+    catchAll: boolean,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the catchall of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressCatchAll({
+        mailAddressId: mailAddressId,
+        data: {
+          active: catchAll,
+        },
+      });
+
+      assertStatus(response, 204);
+    });
+    await process.complete(
+      <Success>
+        The catchall of your mail address was successfully updated.
+      </Success>,
+    );
+  }
+
+  protected async setForwardAddresses(
+    mailAddressId: string,
+    forwardAddresses: string[],
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep(
+      "set the forward addresses of a mail address",
+      async () => {
+        const response =
+          await this.apiClient.mail.updateMailAddressForwardAddresses({
+            mailAddressId: mailAddressId,
+            data: {
+              forwardAddresses: forwardAddresses,
+            },
+          });
+
+        assertStatus(response, 204);
+      },
+    );
+    await process.complete(
+      <Success>
+        The forward addresses of your mail address were successfully updated.
+      </Success>,
+    );
+
+    return;
+  }
+
+  protected async setPassword(
+    mailAddressId: string,
+    password: string,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the password of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressPassword({
+        mailAddressId: mailAddressId,
+        data: {
+          password: password,
+        },
+      });
+
+      assertStatus(response, 204);
+    });
+    await process.complete(
+      <Success>
+        The password of your mail address was successfully updated.
+      </Success>,
+    );
+  }
+
+  protected async setQuota(
+    mailAddressId: string,
+    quota: number,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the quota of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressQuota({
+        mailAddressId: mailAddressId,
+        data: {
+          quotaInBytes: quota,
```

Comment:
This says `quotaInBytes`, but according to the command documentation, the `--quota` flag is "mailbox quota in mebibytes"

Reply:
Which command documentation? If I understand the command handler code correctly, this quotaInBytes is correct.

Reply:
I mean the documentation string of the `--quota` flag. Declared in the `flags` property of this class.

Reply:
Ah, yes i changed it to bytes in the documentation of both the create and update commands.

Reply:
Oooooh, I completely misunderstood that. Yeah, that makes sense. I changed both descriptions back to mebibytes and added mebibyte to byte conversion to `update`.